build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.1.2'
```

Comment:
Please don't add gradle changes with normal issue fixes. So change this file back to the original form.